content/v2.0/influxdb-templates/create.md

```diff
@@ -111,6 +111,70 @@ influx pkg export all \
   --telegraf-configs=00000x0x000X0x0X0
 ```
 
+## Include user-definable resource names
+To let users customize resource names when installing your template, use
+**environment references** in place of names.
+Environment references are replaced with user-defined values when the template is installed.
+
+In your template manifest, replace a [supported resource field](#supported-resource-fields)
```

Comment:
1. How to open/access template manifest...
2. Search for one of the following resource fields to update:
    - metadata.name
    - associations[].name
    - endpointName 
with an `envRef` object that contains a `key` property to reference the user-defined...